test/integration/sync-test.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+buster.testCase('=> Sync', {
+  prepare: function() {
+    this.timeout = 90e3;
+  },
+
+  'can be read': function() {
```

Comment:
Could we add a test that does the initial sync, adds an entry and continuous the sync with the nextSyncUrl, which should then return the just created entry?